storage/src/tests/visiting/visitormanagertest.cpp

```diff
@@ -276,7 +274,7 @@ VisitorManagerTest::getSession(uint32_t n)
             throw vespalib::IllegalStateException(
                     "Timed out waiting for visitor session", VESPA_STRLOC);
         }
-        FastOS_Thread::Sleep(10);
+        std::this_thread::sleep_for(std::chrono::milliseconds(10));
```

Comment:
I find it a lot more readable to use the chrono_literals namespace and 10ms constants.
std::chrono::milliseconds sprinkled around when is just noise.

Reply:
Agreed, will fix this for the next round since these changes have to be re-applied anyway